### PR TITLE
Some more cleanup/modernization

### PR DESCRIPTION
**Pull request by nem035**

### CODE REVIEW

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/javascript/core/flow-control/loops-part-2.md](https://app.enkipro.com/#/public/preview-insight/enkidevs/curriculum/javascript/core/flow-control/loops-part-2.md?ref=30f139eb2e39551c5d4a4fd50822d1e3114f19cd)

File: javascript/core/flow-control/loops-part-2.md [hunk 1]
```diff
@@ -34,84 +19,99 @@ links:
...
-var iterable = [10, 20, 30];
-for (var i of iterable) {
+let arr = [10, 20, 30];
+
+for (let i of arr) {
    console.log(i); //10 20 30
 }
 ```
 
 **Why use for of?**
```
Comment (Stefan-Stojanovic) [inline javascript/core/flow-control/loops-part-2.md]: Suggestion:
"**Why use for of?**" -> "**Why use `for of`?** This can help non-fluent English users.

Comment (Stefan-Stojanovic) [reply javascript/core/flow-control/loops-part-2.md]: Same can be applied to the "**For of**" on line `42`

Comment (nem035) [reply javascript/core/flow-control/loops-part-2.md]:
```suggestion
**Why use `for..of`?**
```

File: javascript/core/flow-control/loops-part-2.md [hunk 2]
```diff
@@ -34,84 +19,99 @@ links:
 ---
 ## Content
 
-*for in* loops allow you to iterate through an objects keys. E.g. the keys below are firstName and lastName:
+`for..in` loops allow you to iterate through an objects keys. E.g. the keys below are `firstName` and `lastName`:
```
Comment (kapnobatai137) [inline javascript/core/flow-control/loops-part-2.md]:
```suggestion
The `for..in` loop allows you to iterate through an object's keys. Consider the following `firstName` and `lastName` keys:
```

File: javascript/core/flow-control/loops-part-2.md [hunk 3]
```diff
@@ -34,84 +19,99 @@ links:
...
-} //firstName John lastName Smith
+for (let key in obj) {
+   console.log(key, obj[key]);
+}
+
+// "firstName" "John"
+// "lastName" "Smith"
 ```
 
 **For of**
```
Comment (kapnobatai137) [inline javascript/core/flow-control/loops-part-2.md]:
```suggestion
### `for...of`
```

File: javascript/core/flow-control/loops-part-2.md [hunk 4]
```diff
@@ -34,84 +19,99 @@ links:
...
 
-*for of* loops are used with iterable objects such as arrays & maps and allow you to easily loop through an iterables values:
+`for..of` loops are used for looping through iterable objects such as arrays:
```
Comment (kapnobatai137) [inline javascript/core/flow-control/loops-part-2.md]:
```suggestion
The `for..of` loops are used for looping through iterable objects such as arrays:
```

File: javascript/core/flow-control/loops-part-2.md [hunk 5]
```diff
@@ -34,84 +19,99 @@ links:
...
 
 ```javascript
-var iterable = [10, 20, 30];
-for (var i of iterable) {
+let arr = [10, 20, 30];
+
+for (let i of arr) {
    console.log(i); //10 20 30
```
Comment (kapnobatai137) [inline javascript/core/flow-control/loops-part-2.md]:
```suggestion
   console.log(i); 
   // 10 20 30
```

File: javascript/core/flow-control/loops-part-2.md [hunk 6]
```diff
@@ -34,84 +19,99 @@ links:
...
 }
 ```
 
-**Why use for of?**
+**Why use `for..of`?**
```
Comment (kapnobatai137) [inline javascript/core/flow-control/loops-part-2.md]:
```suggestion
### Why use `for..of`?
```

File: javascript/core/flow-control/loops-part-2.md [hunk 7]
```diff
@@ -34,84 +19,99 @@ links:
...
-for (var i of iterable) {
-   console.log(i); //10 20 30
+for (let i of iterable) {
+   console.log(i);
 }
+// 10 20 30
 
 ```
 
 **break and continue**
```
Comment (kapnobatai137) [inline javascript/core/flow-control/loops-part-2.md]:
```suggestion
### `break` & `continue`
```

File: javascript/core/flow-control/loops-part-2.md [hunk 8]
```diff
@@ -34,84 +19,99 @@ links:
...
 
-The break statement exits out of a loop and continue will go to the next iteration:
+The `break` statement exits out of a loop and the `continue` statement will skip to the next iteration:
```
Comment (kapnobatai137) [inline javascript/core/flow-control/loops-part-2.md]:
```suggestion
The `break` statement exits out of a loop, while the `continue` statement will skip to the next iteration:
```

File: javascript/core/flow-control/loops-part-2.md [hunk 9]
```diff
@@ -128,8 +128,8 @@ var myArray = [1,2,"3","item"];
...
 
 
-* break
-* stop
-* finish
-* end
+* `break`
+* `stop`
+* `finish`
+* `end`
```
Comment (kapnobatai137) [inline javascript/core/flow-control/loops-part-2.md]: the answers already display as if enclosed by backticks,so we don't need to enclose in backticks anymore

image


```suggestion
* break
* stop
* finish
* end
```

**Pull request merged**